File: we_7_1_how_differential_correction_decays_with_baseline.m
% Integrated Navigation Systems — Chapter 7: GNSS Positioning and Precise Techniques
% Worked Example 7.1 — How differential correction decays with baseline
% Companion code (educational reference implementation; see the book for context).

ppm=0.7e-6; D=30e3; sigma=ppm*D;
fprintf('residual = %.1f cm\n', sigma*100);   % 2.1

File: we_7_1_how_differential_correction_decays_with_baseline.py
# Integrated Navigation Systems — Chapter 7: GNSS Positioning and Precise Techniques
# Worked Example 7.1 — How differential correction decays with baseline
# Companion code (educational reference implementation; see the book for context).

ppm=0.7e-6; D=30e3
print(f'residual = {ppm*D*100:.1f} cm')   # 2.1
